.github/workflows/deploy_to_aws.yml

```diff
@@ -316,9 +316,9 @@ jobs:
           string: ${{ vars.ENV_NAME }}
       - name: Checkout repo
         uses: actions/checkout@v4
-      - name: Run migrations for ${{ vars.ENV_NAME }}
+      - name: Run migrations and other post-deployment tasks for ${{ vars.ENV_NAME }}
         run: |
-          ./deployment/run_cli_task ${{ vars.ENV_NAME }} "migrate,api"
```

Comment:
There is the same line of code in `.github/workflows/db_apply_anonimized.yml`. Should it be updated there as well?

Reply:
@VadimKovalenkoSNF  In my opinion, this command should be executed only in the Deploy to AWS workflow.
I suppose in that case it could be renamed to `post_deployment_to_aws`.

Reply:
I added `post_deployment` command to the post_deploy job of  Apply Anonymized DB workflow.